force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -101,72 +121,29 @@ export default class ReviewSessions extends LightningElement {
-        this.setupDataTableData();
     }
 
-    setupDataTableData() {
-        const FIRSTSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/6/2020";
-        const SECONDSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/7/2020";
-        const THIRDSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/13/2020";
-        const FOURTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/14/2020";
-        const FIFTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/20/2020";
-        const SIXTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/21/2020";
+    // handleAddRows() {
+    //     const dataId = createUUID();
+    //     this._serviceScheduleModel.serviceSessions.push({
+    //         id: dataId,
+    //         Name: "",
+    //         SessionStart__c: "",
+    //         startTime: "",
+    //         endTime: "",
+    //     });
 
-        const DATA = [
-            {
-                id: "1",
-                Name: FIRSTSESSIONNAME,
-                SessionStart__c: "7/6/2020 1:00 PM",
-                SessionEnd__c: "7/6/2020 3:00 PM",
-            },
-            {
-                id: "2",
-                Name: SECONDSESSIONNAME,
-                SessionStart__c: "7/7/2020 1:00 PM",
-                SessionEnd__c: "7/7/2020 3:00 PM",
-            },
-            {
-                id: "3",
-                Name: THIRDSESSIONNAME,
-                SessionStart__c: "7/13/2020 1:00 PM",
-                SessionEnd__c: "7/13/2020 3:00 PM",
-            },
-            {
-                id: "4",
-                Name: FOURTHSESSIONNAME,
-                SessionStart__c: "7/14/2020 1:00 PM",
-                SessionEnd__c: "7/14/2020 3:00 PM",
-            },
-            {
-                id: "5",
-                Name: FIFTHSESSIONNAME,
-                SessionStart__c: "7/20/2020 1:00 PM",
-                SessionEnd__c: "7/20/2020 3:00 PM",
-            },
-            {
-                id: "6",
-                Name: SIXTHSESSIONNAME,
-                SessionStart__c: "7/21/2020 1:00 PM",
-                SessionEnd__c: "7/21/2020 3:00 PM",
-            },
-        ];
-        this.data = DATA;
-    }
-
-    handleAddRows() {
-        const dataId = createUUID();
-        this.data.push({
-            id: dataId,
-            Name: "",
-            SessionStart__c: "",
-            startTime: "",
-            endTime: "",
-        });
-
-        this.data = this.data.slice(0);
-    }
+    //     this._serviceScheduleModel.serviceSessions = this._serviceScheduleModel.serviceSessions.slice(0);
+    // }
 
     handleDelete(event) {
-        this.data = this.data.filter(element => element.id !== event.detail.row.id);
+        this._serviceSessions = this.serviceSessions.filter(session =>
+            Object.values(this._serviceScheduleModel.sessionFields).reduce(
+                (matchesSoFar, field) =>
+                    matchesSoFar &&
+                    session[field.apiName] !== event.detail.row[field.apiName],
```

Comment:
Running into a couple issues if I intentionally clear out the ending date/time on page 1. When I delete the session everything is filtered out. I also noticed a warning in the console from the formatted date time component about it being provided an invalid date.
![InvalidDate](https://user-images.githubusercontent.com/450473/94024219-c4568d00-fd7c-11ea-8c83-f477389c4462.png)